In [1]:
def chi():
    WA=90
    BA=30
    NA=30

    WB=60
    BB=50
    NB=40

    WC=104
    BC=51
    NC=45

    WD=95
    BD=20
    ND=35

    totalWhite=WA+WB+WC+WD
    totalBlue=BA+BB+BC+BD
    totalNo=NA+NB+NC+ND

    totalA=WA+BA+NA
    totalB=WB+BB+NB
    totalC=WC+BC+NC
    totalD=WD+BD+ND

    completTotal=totalWhite+totalBlue+totalNo

    expeectedWhiteA= totalA *(totalWhite/completTotal)
    expeectedWhiteB= totalB *(totalWhite/completTotal)
    expeectedWhiteC= totalC *(totalWhite/completTotal)
    expeectedWhiteD= totalD *(totalWhite/completTotal)

    expeectedBlueA= totalA *(totalBlue/completTotal)
    expeectedBlueB= totalB *(totalBlue/completTotal)
    expeectedBlueC= totalC *(totalBlue/completTotal)
    expeectedBlueD= totalD *(totalBlue/completTotal)

    expeectedNoA= totalA *(totalNo/completTotal)
    expeectedNoB= totalB *(totalNo/completTotal)
    expeectedNoC= totalC *(totalNo/completTotal)
    expeectedNoD= totalD *(totalNo/completTotal)

    

    hypothesisAWhite = ((WA-expeectedWhiteA)**2)/expeectedWhiteA
    hypothesisBWhite = ((WB-expeectedWhiteB)**2)/expeectedWhiteB
    hypothesisCWhite = ((WC-expeectedWhiteC)**2)/expeectedWhiteC
    hypothesisDWhite = ((WD-expeectedWhiteD)**2)/expeectedWhiteD
    hypothesisTotalWhite=hypothesisAWhite+hypothesisBWhite+hypothesisCWhite+hypothesisDWhite

    hypothesisABlue = ((BA-expeectedBlueA)**2)/expeectedBlueA
    hypothesisBBlue = ((BB-expeectedBlueB)**2)/expeectedBlueB
    hypothesisCBlue = ((BC-expeectedBlueC)**2)/expeectedBlueC
    hypothesisDBlue = ((BD-expeectedBlueD)**2)/expeectedBlueD
    hypothesisTotalBlue=hypothesisABlue+hypothesisBBlue+hypothesisCBlue+hypothesisDBlue

    hypothesisANo = ((NA-expeectedNoA)**2)/expeectedNoA
    hypothesisBNo = ((NB-expeectedNoB)**2)/expeectedNoB
    hypothesisCNo = ((NC-expeectedNoC)**2)/expeectedNoC
    hypothesisDNo = ((ND-expeectedNoD)**2)/expeectedNoD
    hypothesisTotalNo=hypothesisANo+hypothesisBNo+hypothesisCNo+hypothesisDNo

    #Answer comes out as 24.57120
    hypothesisTotal=hypothesisTotalWhite+hypothesisTotalBlue+hypothesisTotalNo
     
    formatted_hypothesis = "{:.1f}".format(hypothesisTotal)
    print(formatted_hypothesis)


   
    
chi()

24.6


White stands for white collar
Blue stands for blue collar
No stands for no collar

There is a other way of doing it if you use scopy.stats
I think you have to use https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.combine_pvalues.html#scipy.stats.combine_pvalues this function. 
I couldn't figure it the parameter so i did it the long way.
I did verify that Chi-squared value based on it is approximately 24.6. 